docs/paper/contributing/events.md

```diff
@@ -0,0 +1,26 @@
+---
+slug: /contributing/events
+---
+
+# Events
+There are several requirements for events in the Paper API. Note that while not all existing events may follow these
```

Comment:
this is the bit that shows up in the embed. Maybe make a slightly more user friendly sentence for this, move this down and the note could even go in an amondition